test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -309,6 +309,10 @@ Describe "Get-Help should find pattern help files" -Tags "CI" {
         )
         $command.Invoke() | Should -Be $result
     }
+
+    It "Get-Help should not fail searching for class help" {
+        Get-Help -Category Class | Should -BeNullOrEmpty
```

Comment:
The test can take a very long time. Can we use `-Name` parameter?

Reply:
Because we don't ship any script classes, this operation doesn't take very long.  It's only hit when the test modules are in your PSModulePath specifically the PSLogItem class.  If you run this test directly w/o using build.psm1 which adds the test modules to your path, this test will fail as it wouldn't find `PSLogItem` class.  I think for now we can just leave it as-is.

Reply:
Maybe add a class in the test to exclude problems in future? 
I run on Windows PowerShell - it is very long.

Reply:
@iSazonov that's a reasonable request.  I was testing on macOS and it's pretty fast.